### DISCUSSION TITLE
Add alt tag to carousel

### DISCUSSION THREAD

**Discussion by dotlineX** [nunocoracao/blowfish#617] [Ideas]
Hi there :wave:

I wanted to start off by saying that this theme is great!

Recently, I noticed that the figure shortcode supports alt tags, but the carousel shortcode does not. This can cause some warnings with SEO tools.

This issue can be resolved by adding an alt tag with `carousel image {{$num}}` . I'm sure this is not the best solution but its one way to do it I guess 🤷‍♂️ 

```html
{{ $num := 0 }}
{{ range $images }}

<div class="relative float-left -mr-[100%] {{ if not (eq $num 0) }} hidden {{ end }} w-full transition-transform duration-[{{ $interval }}ms] ease-in-out motion-reduce:transition-none"
     data-te-carousel-item {{ if eq $num 0 }} data-te-carousel-active {{ end }}>
    <div class="ratio-{{ $aspect }} single_hero_background">
        <img src="{{ . }}" class="block absolute top-0 object-cover w-full h-full" alt="carousel image {{$num}}"/>
    </div>
</div>

{{ $num = add $num 1 }}
{{ end }}
```
If you agree with this solution, I'm willing to open a pull request to implement it.
Any feedback is welcome!

Thank you for your time, and have a great day!

**Comment by nunocoracao** [comment]
@dotlineX sounds like a quick good addition. And I would really welcome the PR. If you can't do it, please open the ticket for the fix. 
Thanks
